src/parcsr_mv/par_csr_matvec.c

```diff
@@ -849,3 +857,49 @@ hypre_ParCSRMatrixMatvec_FF( HYPRE_Complex       alpha,
 
    return ierr;
 }
+
```

Comment:
Maybe it should be put in `par_csr_matop.c`. In `par_csr_matop_device.c`, there is one `hypre_ParCSRDiagScale` doing similar thing (with beta==0.0, alpha=1.0, also for CPU), so we should combine them.

Reply:
I think we need to decide whether maintaining a function to compute `y = inv(A_D)*x` in the `ParCSRMatrix` class or having only a `ParVector` function that does `y = alpha*z.*x + beta*b` (element-wise multiplication for `z` and `x`) that could be used for computing the former quantity. I think @rfalgout wanted to bring this for discussion.

Reply:
@victorapm Either way is fine. We have device version for both of them. So I would like to combine with the host function you added.